src/server/tran_server.hpp

```diff
@@ -102,11 +131,13 @@ class tran_server
 	const std::string get_channel_id () const;
 
       protected:
-	connection_handler (cubcomm::channel &&chn, tran_server &ts, request_handlers_map_t &&request_handlers);
+	connection_handler (cubcomm::channel &&chn, page_server_node &node, tran_server &ts,
+			    request_handlers_map_t &&request_handlers);
 
 	virtual request_handlers_map_t get_request_handlers ();
 
       protected:
+	page_server_node &m_node; // in tran_server::m_node_vec;
```

Comment:
with the new approach (as described above) this will be a
```suggestion
	cubcomm::node m_conn_node; // or, can even be a const reference to the entry in tran_server::m_connection_list
```